layouts/black-lives-matter.hbs

```diff
@@ -0,0 +1,207 @@
+      <li>Jamarion Robinson</li>
+      <li>Jamee Johnson</li>
+      <li>Jemel Roberson</li>
+      <li>Jerame Reid</li>
+      <li>Jimmy Atchison</li>
+      <li>John Crawford III</li>
+      <li>Jonathan Ferrell</li>
+      <li>Jonathan Hart</li>
+      <li>Jordan Baker</li>
+      <li>Jordan Edwards</li>
+      <li>Julius Johnson</li>
+      <li>Kajieme Powell</li>
+      <li>Keith Childress</li>
+      <li>Keith Lamont Scott</li>
+      <li>Kendrec McDade</li>
+      <li>Kwame Jones</li>
+      <li>Laquan McDonald</li>
+      <li>Larry Jackson Jr</li>
+      <li>Manuel Loggins Jr</li>
+      <li>Mario Woods</li>
+      <li>Maurice Granton</li>
+      <li>Michael Brown</li>
+      <li>Michael Dean</li>
+      <li>Mike Ramos</li>
+      <li>Miles Hall</li>
+      <li>Natosha McDade</li>
+      <li>Patrick Harmon</li>
+      <li>Philando Castile</li>
+      <li>Quintonio LeGrier</li>
+      <li>Ramarley Graham</li>
+      <li>Robert Lawrence White</li>
+      <li>Rumain Brisbon</li>
+      <li>Ryan Twyman</li>
+      <li>Samuel David Mallard</li>
+      <li>Samuel DuBose</li>
+      <li>Sandra Bland</li>
+      <li>Sean Reed</li>
+      <li>Stephon Clark</li>
+      <li>Steven Demarco Taylor</li>
+      <li>Tamir Rice</li>
+      <li>Terence Crutcher</li>
+      <li>Terrance Franklin</li>
+      <li>Terrence Sterling</li>
+      <li>Tony McDade</li>
+      <li>Tony Robinson</li>
+      <li>Trayvon Martin</li>
+      <li>Victor White III</li>
+      <li>Walter Scott</li>
+      <li>Wendell Allen</li>
+      <li>William Green</li>
+      <li>Willie McCoy</li>
+      <li>Yassin Mohamed</li>
+      <li><a href="https://mappingpoliceviolence.org/unarmed">and</a> <a href="https://mappingpoliceviolence.org/unarmed2014">countless</a> <a href="https://apps.npr.org/documents/document.html?id=6933593-NPR-CodeSwitch-Saytheirnameslistv3">more</a> names not listed here whose lives were taken by violence and brutality</li>
+    </ul>
+  </section>
+
+  <section>
+    <p>Where you can donate to with your wallet:</p>
+    <ul>
+      <li>Donate to the <a href="https://www.gofundme.com/f/georgefloyd">Official George Floyd Memorial Fund</a>.</li>
+      <li>Donate to the <a href="https://www.gofundme.com/f/i-run-with-maud">I Run With Maud</a> fundraiser for Ahmaud Arbery.</li>
+      <li>Debate to the <a href="https://www.gofundme.com/f/justice-for-mike-ramos">Justice for Mike Ramos</a> fundraiser.</li>
+      <li>Find and donate to bail fund via the <a href="https://www.communityjusticeexchange.org/nbfn-directory">National Bail Fund Network</a>.</li>
+      <li><a href="https://www.facebook.com/reclaimtheblock/">Reclaim the Block</a> has published a <a href="https://docs.google.com/document/d/1yLWGTQIe3967hdc9RSxBq5s6KKZHe-3_mWp5oemd7OA/preview?pru=AAABcpUiX3k*Y6Q4I6UBtkH3lLz9GVLg0A">Google Doc of organizations to donate to.</a></li>
+    </ul>
+    <p>We will be updating this list periodically.</p>
+  </section>
+
+  <section id="our-support">
```

Comment:
```suggestion
  <section id="our-support">

  <p>We are currently planning to have mentorship for the duration of the upcoming Collaborator Summit for Black folks to help mentor and onboard. If you are interested in this, please feel free to send us an email at <a href="mailto:blacklivesmatter@nodejs.org">blacklivesmatter@nodejs.org</a> or feel free to DM us at <a href="https://twitter.com/nodejs">@nodejs</a> on Twitter. We will update y'all with more context on how we're going to be proceeding with this as we finalize the details.</p>
```

Reply:
Also, if folks are going to ask about the email address: @brianwarner is/has creating/created it.

Reply:
Definitely have a number of issues with the proposed new text. (e.g. `folks`, `y'all`).

For this entire section, perhaps the following:

```
In Node.js, we recognize that there is still an incredible amount of
work to be accomplished to address the injustices and inqualities that
exist, not only in society as a whole but within the technology industry,
and even within our own project. To this point, we have not done enough
and recognize that we need to do more. We commit to supporting the Black
members of the community by using our collective platforms to amplify your
words, or by helping you in your careers, contributing to open source, and
seeking other opportunities to help.

Beginning with our upcoming Collaborator Summit event, and extending
beyond, we will be offering mentorship to Black members of the community
who wish to become contributors to the Node.js project. If you are interested
in this, please send us an email at <a href="mailto:blacklivesmatter@nodejs.org">blacklivesmatter@nodejs.org</a>
or direct message us at <a href="https://twitter.com/nodejs">@nodejs</a>
on Twitter. We will post more information on how we will be proceeding
with this effort as we finalize the details.
```

Reply:
```suggestion
  <section id="our-support">

  <p>In Node.js, we recognize that there is still an incredible amount of work to be accomplished to address the injustices and inequalities that exist, not only in society as a whole but within the technology industry, and even within our own project. To this point, we have not done enough and recognize that we need to do more. We commit to supporting the Black members of the community by using our collective platforms to amplify your words, or by helping you in your careers, contributing to open source, and seeking other opportunities to help.</p>

 <p>Beginning with our upcoming Collaborator Summit event, and extending beyond, we will be offering mentorship to Black members of the community who wish to become contributors to the Node.js project. If you are interested in this, please send us an email at <a href="mailto:blacklivesmatter@nodejs.org">blacklivesmatter@nodejs.org</a> or direct message us at <a href="https://twitter.com/nodejs">@nodejs</a> on Twitter. We will post more information on how we will be proceeding with this effort as we finalize the details both for the period of the Collaborator summit and beyond that single event.</p>
```

Reply:
@jasnell formatted for HTML and added a slight bit at the end to have more clarity.

Reply:
Looks good to me.

Reply:
Re my comment below, you dance around your complicity. You need to explicitly say "We recognise that [our entire team/90%, 10/12] is white/[white-presenting], and [X percent] male/masculine-presenting." (or whatever your demographics truly are).

Reply:
If James's approach is appealing, here's how I'd edit it:

```suggestion
<p>There is an incredible amount of work to do to address injustices and inequalities. In society, the technology industry, and our own project, we have not done enough. We need to do more. We commit to supporting the Black members of the community. We will use our collective platforms to amplify your words.  We will help you in your careers. We will help you in contributing to open source. And we seek other opportunities to help.</p>

<p> We are offering mentorship to Black members of the community who wish to become contributors to the Node.js project. If interested, please send an email to <a href="mailto:blacklivesmatter@nodejs.org">blacklivesmatter@nodejs.org</a> or a direct message to <a href="https://twitter.com/nodejs">@nodejs on Twitter</a>.</p>
```